### PR TITLE
DOP Test & update integration-snippets with recent mTLS change. Add ruby

### PR DESCRIPTION
**Pull request by oschvr**

### CODE REVIEW

**Comment by oschvr** [comment]
Screenshot 2022-06-17 at 2 44 46 pm
Screenshot 2022-06-17 at 2 49 16 pm
Screenshot 2022-06-17 at 2 47 13 pm
Screenshot 2022-06-17 at 3 09 22 pm

File: mtls-examples/ruby/mtls.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require 'openssl'
+require 'net/http'
+
+options = {
+  use_ssl: true,
+  verify_mode: OpenSSL::SSL::VERIFY_PEER,
+  cert: OpenSSL::X509::Certificate.new(File.read('cert.crt')),
+  key: OpenSSL::PKey::RSA.new(File.read('key.pem')),
+  ca_file: 'ca.pem'
```
Comment (amroessam) [inline mtls-examples/ruby/mtls.rb:9]: Needs a file probably, not just a filename

File: mtls-examples/go/mtls.go [hunk 2]
```diff
@@ -11,7 +11,7 @@ import (
 
 var (
 	// cert file MUST be a concatenation of certificate provided by Lean AND the certificate chain!!!
-	certFile = flag.String("cert", "cert_and_cert_chain.crt", "Your application certificate provided by Lean concatenated with cert chain")
+	certFile = flag.String("cert", "cert.crt", "Your application certificate provided by Lean concatenated with cert chain")
```
Comment (amroessam) [inline mtls-examples/go/mtls.go:14]: I think the previous one was working, one of our clients said so. Can we confirm?

**Review by amroessam — CHANGES_REQUESTED** [review]

File: mtls-examples/ruby/mtls.rb (hunk 1 above)
Comment (oschvr) [reply mtls-examples/ruby/mtls.rb:9]: It seems that for `ca_file` property of Net::HTTP class, only a path is expected....

An example here 
https://www.rubydoc.info/stdlib/net/Net%2FHTTP:ca_file

And here
https://gist.github.com/floehopper/714559 

And the docs
https://www.rubydoc.info/stdlib/net/Net%2FHTTP:ca_file

There is no Class for `ca_file` in `OpenSSL`. 
Also tried with method `File.read()` without success.

File: mtls-examples/go/mtls.go (hunk 2 above)
Comment (oschvr) [reply mtls-examples/go/mtls.go:14]: This is just a name change. 
Keeping it `cert_and_cert_chain.crt` might be misleading as with the change to mTLS certificate issuing from developer portal, we are now giving our clients the *cert AND the chain* in the single file `cert.crt`

File: mtls-examples/ruby/mtls.rb (hunk 1 above)
Comment (oschvr) [reply mtls-examples/ruby/mtls.rb:9]: Screenshot 2022-07-11 at 5 29 24 pm

Comment (amroessam) [reply mtls-examples/ruby/mtls.rb:9]: 👍🏽

**Review by amroessam — APPROVED** [review]

**Pull request merged**